packages/example-embedder/src/lib/internal/targetToExampleMap.ts

```
@@ -0,0 +1,31 @@
+  return files.reduce<Record<string, ExampleMap>>((accumulator, file) => {
+    const [firstLine, ...code] = file.content.split("\n");
+    const targetPaths = firstLine?.match(/\/\/ @example\s+(.+)/)?.[1]?.split(",");
+
+    if (!targetPaths) {
+      return accumulator;
+    }
+
+    return targetPaths.reduce((intermediateResult, targetPath) => {
+      const trimmedPath = targetPath.trim();
+      return {
+        ...intermediateResult,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)